spec/components/blacklight/search_context/server_item_pagination_component_spec.rb

```diff
@@ -23,6 +23,18 @@
     end
   end
 
+  context 'when there is exactly one search result with no next or previous document' do
```

Comment:
What if someone goes directly to a show page without any session history.  Will they still see this?

Reply:
After the most recent changes, I went directly to an item page url in a separate browser and that did not show the "1 of 1" content.